modules/storage/cosmos-db/README.md

```diff
@@ -68,11 +68,10 @@ It allows for the creation of a new Cosmos DB account or use of an existing one
 An example of how to deploy Azure Cosmosdb Account with _Apache Cassanda backend_ using the minimum required parameters.
 
 ```bicep
-module cosmosDbAccount 'br/public:storage/cosmos-db:2.0.0' = {
+module cosmosDbAccount 'br/public:storage/cosmos-db:0.0.1' = {
```

Comment:
```suggestion
module cosmosDbAccount 'br/public:storage/cosmos-db:2.0.1' = {
```

the version should always end in at least X.X.1 (we never use X.X.0)

Reply:
updated